src/transform.ts

```diff
@@ -177,12 +168,54 @@ function concatArrays(...arrs: ArrayLike<number>[]) {
   return c
 }
 
-function extractAndNormaliseTransaction(
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+const objectToMap = (object: Record<string, any>): Map<string, unknown> => {
```

Comment:
The naming of this function could probably be improved - it has algorand specific field names in there so "objectToMap" doesn't really describe what it's doing.

Reply:
(I get it's private, but still probably a good idea to make the name correctly descriptive)